modules/apps/web-experience/export-import/export-import-api/src/main/java/com/liferay/exportimport/staged/model/repository/StagedModelRepository.java

```diff
@@ -49,16 +84,32 @@ public default T fetchMissingReference(String uuid, long groupId) {
 	public List<T> fetchStagedModelsByUuidAndCompanyId(
 		String uuid, long companyId);
 
+	/**
+	 * If possible, use return _yourEntityLocalService.getExportActionableDynamicQuery(portletDataContext); as the implementation.
+	 * @param portletDataContext
+	 * @return
```

Comment:
@akosthurzo can you also add the info for all `@param` and `@return` tags in this new Javadoc? Most of them are probably pretty self-explanatory, but having descriptions for them removes any doubt readers may have.